manifests/kube/kube@0.0.1.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "kube",
+  "description": "A plugin to manage spin apps",
```

Comment:
Do we want the  descriptions to match?

Reply:
I’m okay with having the descriptions to match. However I’m also happy to leave it as-is for historical purposes.

Reply:
I am good with either as well.

Reply:
I'll leave it as-is for historical purposes